### ISSUE TITLE
We've created a version of v7.0.0 which works with Python 3.6

### ISSUE THREAD

**Issue by infectious** [googleads/google-ads-python#331]
Hello,

On 25th Sep 2020 we asked the question if a version of google-ads be created that talks to the v3 (or higher) Google Ads API and works on Python 3.6.6 - https://github.com/googleads/google-ads-python/issues/319

The answer was that this can't be done because the package uses a dynamic import mechanism that relies on a module-level getattr method, which is only available in Python 3.7.0+ - which was fair enough.

Since then we've created a version of the package that imports the pep562 package - https://pypi.org/project/pep562/ - to backport the __getattr__ function from pep562 into older versions of Python. It only took a few fairly basic changes to the package to get this working, and we've successfully tested it on Python 3.6.x. Our changes will detect if Python is 3.7+ and use the built-in __getattr__ if so, and if it's lower than 3.7 it will use the pep562 module.

In order for us to use this in our Airflow environment running under Google Composer, we need the functionality to be present in the publically available version of google-ads-python in pypi. Composer's limitations mean we cannot import it any other way e.g. from a private repo (very frustrating!).

So, would you be interested in accepting these changes as an official update to your package?

Thanks,
Harry

**Comment by davedavis** [comment]
We're about to migrate to Compose from a self managed airflow instance and didn't realize that 3.6 is the highest supported version on Compose. Bit of a tricky situation now, wait for this as a PR, wait for Compose to bump or use your fork/branch. None of these are good options. 

The only sensible option I feel now is to containerize Google Ads applications while we're waiting and move when there's more certainty.

**Comment by infectious** [comment]
Yes we investigated the containerisation route e.g. Kube operator, but for us it is goes against the reason we've used Composer, which is to reduce the devops load and knowledge needed (Kube knowledge is low in the team). We'd rather solve this in code, so to speak. But this is just our setup - your situation may be entirely different.

Hopefully our change will be accepted as it's really robust, given that we're just using the pep562 package which has been out there for a while now. Our changed to google_ads itself are entirely around including that package rather than anything deeper.
